spec/acceptance/users_spec.rb

```diff
@@ -0,0 +1,100 @@
+require "rspec_api_documentation/dsl"
+
+RSpec.resource "Users" do
+  let(:authenticated_user) do
+    create(:confirmed_user)
+  end
+
+  let(:auth_token) do
+    Knock::AuthToken.new(payload: { sub: authenticated_user.id }).token
+  end
+
+  shared_context "user parameters" do
+    parameter "first_name", <<-DESC, scope: :user, required: true
+      The first name of the user.
+    DESC
+    parameter "email", <<-DESC, scope: :user, required: true
+      The email of the user.
+    DESC
+    parameter "last_name", <<-DESC, scope: :user, required: true
+      The last name of the user.
+    DESC
+    parameter "password", <<-DESC, scope: :user, required: true
+      The password of the user.
+    DESC
+    parameter "city", <<-DESC, scope: :user, required: true
+      The city of the user.
+    DESC
+    parameter "state", <<-DESC, scope: :user, required: true
+      The state of the user.
+    DESC
+    parameter "country", <<-DESC, scope: :user, required: true
+      The country of the user.
+    DESC
+  end
+
+  post "/users" do
+    include_context "user parameters"
+
+    let(:first_name) { "Nate" }
+    let(:email)      { "admin@test.com" }
+    let(:last_name)  { "Pauzenga" }
+    let(:city)       { "Annapolis" }
+    let(:state)      { "MD" }
+    let(:country)    { "USA" }
+    let(:password)   { "helloworld" }
+
+    example_request "POST /users" do
+      expect(status).to eq 201
+      json_response = JSON.parse(response_body)
+      expect(json_response["jwt"]).to include("payload")
+    end
+  end
+
+  patch "/users/:id" do
```

Comment:
@Enriikke so here we ran into problems. 

First, I'm giving my `UpdateUser` Interactor the `current_user` right off the bat and not looking for a User based on the `id` param. This means that you could call this endpoint with any `id` and it will still only update you, the `current_user`. `/users/"awesome"` works just as well as `users/1`. On the one hand it makes it pretty secure in that a user can definitely not update someone else's info. On the other hand, I don't think this is how you'd expect this to work.

Also, perhaps we could talk about our User validations on update on the next call.